components/style/gecko_bindings/bindings.rs

```diff
@@ -518,6 +518,12 @@ extern "C" {
     pub fn Gecko_DestroyClipPath(clip: *mut StyleClipPath);
 }
 extern "C" {
+    pub fn Gecko_ResetFilters(effects: *mut nsStyleEffects, len: usize);
+}
+extern "C" {
+    pub fn Gecko_CopyFiltersFrom(aSrc: &nsStyleEffects, aDest: *mut nsStyleEffects);
```

Comment:
hmm... This seems to be written manually, given bindgen doesn't generate references? I think you should actually regenerate the bindings, you can do that easily without regenating the structs running:

```
./regen.py --target bindings /path/to/objdir
```